examples/hil.cfg

```diff
@@ -134,3 +134,6 @@ save = True
 # Same behaviour as the dell switch. Set `save` to False to stop the switch
 # from writing to flash memory.
 save = True
+
+[hil.ext.switches.dellnos9]
+save = True
```

Comment:
>  Is there a reason we would want to have this on for some switches but not others?

Reply:
>  Is there a reason we would want to have this on for some switches but not others?

If the switches are stacked (aka logical chassis mode), then switches are saving automatically and the calls to `save_running_config` would fail. 
E.g.
```
sw0# copy running-config startup-config 
This operation is not supported in logical chassis mode.
```

Of course, we could handle a scenario like this in code, but I think the burden should be on hil-admins (who will know what switches they have stacked or not) to do this setup correctly because HIL would unnecessarily try and fail in this case.

Reply:
Point being, some type of switches could be stacked while other type might not be, and I think admins should use the hil config file to tell HIL about it.